include/igl/direct_delta_mush.h

```diff
@@ -0,0 +1,87 @@
+// This file is part of libigl, a simple c++ geometry processing library.
+//
+// Copyright (C) 2020 Xiangyu Kong <xiangyu.kong@mail.utoronto.ca>
+//
+// This Source Code Form is subject to the terms of the Mozilla Public License
+// v. 2.0. If a copy of the MPL was not distributed with this file, You can
+// obtain one at http://mozilla.org/MPL/2.0/.
+#ifndef IGL_DIRECT_DELTA_MUSH_H
+#define IGL_DIRECT_DELTA_MUSH_H
+
+#include "igl_inline.h"
+
+#include <Eigen/Core>
+#include <Eigen/Sparse>
+#include <Eigen/Geometry>
+#include <vector>
+
+namespace igl {
+  // Computes Direct Delta Mesh Skinning (Variant 0) from "Direct Delta Mush
+  // Skinning and Variants"
+  //
+  // Inputs:
+  //   V  #V by 3 list of rest pose vertex positions
+  //   F  #F by 3 list of triangle indices into rows of V
+  //   C  #C by 3 list of rest pose bone endpoint positions
+  //   E  #T by 2 list of bone edge indices into rows of C
+  //   T  #T list of bone pose transformations
+  //   Omega #V by #T*10 list of precomputated matrix values
+  // Outputs:
+  //   U  #V by 3 list of output vertex positions
+  template <
+    typename DerivedV,
+    typename DerivedF,
+    typename DerivedC,
+    typename DerivedE,
+    typename DerivedOmega,
+    typename DerivedU>
+  IGL_INLINE void direct_delta_mush(
+    const Eigen::MatrixBase<DerivedV> & V,
+    const Eigen::MatrixBase<DerivedF> & F,
+    const Eigen::MatrixBase<DerivedC> & C,
```

Comment:
Are `C` and `E` used in the end? If not, then let's remove them from (both) prototypes.

Reply:
They are not used. I will remove them.